jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -626,6 +627,39 @@ boolean isSkipped() {
     return skip;
   }
 
+  /**
+   * Return false if the `jib.comntainerized` property is specified and does not match this
```

Comment:
two typos in `jib.comntainerized`. Removed the trailing `d` and the extraneous `m` in the middle.

Reply:
Jeepers, this was contaminated.